### PR TITLE
Fix travis build

### PR DESCRIPTION
**Pull request by stefina**

### CODE REVIEW

File: bin/travis-build.bash [hunk 1]
```diff
@@ -5,12 +5,14 @@ echo "This is travis-build.bash..."
...
-sudo apt-get install postgresql-$PGVERSION solr-jetty libcommons-fileupload-java:amd64=1.2.2-1
+sudo apt-get install solr-jetty libcommons-fileupload-java
 
 echo "Installing CKAN and its Python dependencies..."
 git clone https://github.com/ckan/ckan
 cd ckan
-git checkout $CKAN_BRANCH
+export latest_ckan_release_branch=`git branch --all | grep remotes/origin/release-v | sort -r | sed 's/remotes\/origin\///g' | head -n 1`
+echo "CKAN branch: $latest_ckan_release_branch"
+git checkout $latest_ckan_release_branch
```
Comment (amercader) [inline bin/travis-build.bash]: I know that's what the extension template will output, but with this change we are losing the testing across different CKAN versions (`CKAN_VERSION` env var). For general purpose extensions like this one it's worth testing against different versions to ensure they work as expected. Here's an [example](https://github.com/frictionlessdata/ckanext-validation/blob/master/bin/travis-build.bash#L14:L19).

**Comment by amercader** [comment]
@stefina if you are currently using it, would you have time to help maintain this extension, reviewing and merging this and other PRs?

File: bin/travis-build.bash (hunk 1 above)
Comment (torfsen) [reply bin/travis-build.bash]: That's something we ran into for ckanext-pages, too. See ckan/ckanext-pages#62.

**Comment by stefina** [comment]
@amercader Yeah, I can do that, not sure if I have knowledge about everything here, but at least we are using it ;)

File: bin/travis-build.bash (hunk 1 above)
Comment (stefina) [reply bin/travis-build.bash]: thanks for the hint @amercader and @torfsen !
I'll have a look into that probably end of this week.

**Comment by stefina** [comment]
@amercader I finally had some time to fix the Travis-Build. Would you be able to take a look at this, if you have time?

File: ckanext/showcase/tests/test_plugin.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from routes import url_for
+from ckan.lib.helpers import url_for
```
Comment (amercader) [inline ckanext/showcase/tests/test_plugin.py]: For extra points next time use `toolkit.url_for` :ok_hand:

**Comment by amercader** [comment]
Many thanks @stefina!

File: ckanext/showcase/tests/test_plugin.py (hunk 2 above)
Comment (stefina) [reply ckanext/showcase/tests/test_plugin.py]: will do, thanks for the hint and for merging :)

**Pull request merged**